types.go

```diff
@@ -202,6 +202,12 @@ const (
 	PinByName
 )
 
+// LoadPinOptions control how a pinned object is loaded.
+type LoadPinOptions struct {
+	// Flags may be BPF_F_RDONLY or BPF_F_WRONLY.
+	Flags uint32
```

Comment:
Why does the field need to be part of a struct?

Reply:
I doesn't necessarily. It's in a struct because that allows extending the API in the future, without having to break compatibility.